lib.sh

```diff
@@ -1079,7 +1132,8 @@ rm -f releases
 # Use it for functions like download_static_script
 download_script() {
     rm -f "${SCRIPTS}/${2}.sh" "${SCRIPTS}/${2}.php" "${SCRIPTS}/${2}.py"
-    if ! { curl_to_dir "${!1}" "${2}.sh" "$SCRIPTS" || curl_to_dir "${!1}" "${2}.php" "$SCRIPTS" || curl_to_dir "${!1}" "${2}.py" "$SCRIPTS"; }
+    if ! { curl_to_dir "${!1}" "${2}.sh" "$SCRIPTS" || curl_to_dir "${!1}" "${2}.php" "$SCRIPTS" \
+    || curl_to_dir "${!1}" "${2}.py" "$SCRIPTS" || curl_to_dir "${!1}" "${2}" "$SCRIPTS"; }
```

Comment:
this addition is needed to be able to download .dat files.

Reply:
Hmm, I'm not so sure about this... Please make a separate function for dat files, or add.dat, otherwise anything in `$2` can be downloaded.

Reply:
When thinking about it. Make a separate function. .dat files are not scripts, and the text expects the file to be a script.

Reply:
done